src/css/components/_img-content.scss

```diff
@@ -43,3 +43,38 @@
      }
   }
 }
+
+.afo {
+  h2 {
+    text-align: center;
+  }
+  p {
+    text-align: center;
+  }
+  &__product {
+    display: grid;
```

Comment:
We'll need ALL grids to be run through this: https://autoprefixer.github.io

Particularly adding in the -ms- prefixes to make sure it works. It'll need to be ran from the compiled version. Then we just prefix where needed. 

If this is a multi-repeat, we need to add ":nth-child" for each ROW. Alternative -- mobile version just becomes display block using this media query:

```
@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
      /* IE10+ CSS styles go here */
    }
```

**Of Note:** grid-*-gaps will NEVER render for IE.